linera-service/src/database_tool.rs

```diff
@@ -192,3 +182,21 @@ async fn main() -> Result<(), anyhow::Error> {
     tracing::info!("Successful execution of linera-db");
     Ok(())
 }
+
+#[tokio::main]
+async fn main() {
+    let env_filter = tracing_subscriber::EnvFilter::builder()
+        .with_default_directive(tracing_subscriber::filter::LevelFilter::INFO.into())
+        .from_env_lossy();
+    tracing_subscriber::fmt()
+        .with_writer(std::io::stderr)
+        .with_env_filter(env_filter)
+        .init();
+
+    let options = DatabaseToolOptions::from_args();
+    let error_code = match evaluate_options(options).await {
```

Comment:
Probably no need to define this function

Reply:
Could you explain? In the current design, we must catch the error to identify what is happening.

If we used `Result<(), anyhow::Error>` as before, we have nontrivial error codes. But the problem is that they would overlap.
For example, I just tested with `linera-db check_existence` and I obtained the error code `1` which is exactly the one we want for the absence of a table. So, we have an error code overlap and so I think we have to catch the `Err` case.

Reply:
Never mind. I'm fine with the current function.